### PR TITLE
etcd-operator: update k8s descriptors with cilium 1.2.2

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5583)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by ianvernon** [comment]
@aanm  I'm about to cut the v1.2.3 release. Do you want me to include this step (updating versions to v1.2.3) as a commit in that branch as well?

File: examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml [hunk 1]
```diff
@@ -81,181 +100,222 @@ spec:
...
-                  name: cilium-config
-                  optional: true
-            - name: "CILIUM_MONITOR_AGGREGATION_LEVEL"
-              valueFrom:
-                configMapKeyRef:
-                  key: monitor-aggregation-level
-                  name: cilium-config
-                  optional: true
-          livenessProbe:
+      - image: docker.io/cilium/cilium:latest
```
Comment (ianvernon) [inline examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml:131]: The commit title is inaccurate given this change. Specify in the commit description that we are changing the YAML for developers to use when testing this functionality to use latest, despite this being the v1.2 branch.

**Review by ianvernon — DISMISSED** [review]

**Comment by aanm** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

**Pull request merged**